requirements.txt

```diff
@@ -1,6 +1,6 @@
 lxml>=3.2.4
 # ncclient version 0.6.10 has issues with PyEZ(junos-eznc) and needs to be avoided
-ncclient==0.6.13
+ncclient==0.6.15
```

Comment:
Really should make this a `ncclient>=0.6.15` (to try avoid pinning to a specific version of ncclient).

Reply:
Hi @ktbyers 
    Thanks vey much for the review comments .
   I will update the fix .

Thanks